mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go

```diff
@@ -0,0 +1,245 @@
+package mongodbatlas
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceMongoDBAtlasAdvancedClusterResourceV0() *schema.Resource {
```

Comment:
[q] What is the `V0` in the name stand for?

Reply:
V0 schema part of state migration function to move old schema value to new one bi_connector becomes bi_connector_config in this release breaking change